python/sidecar/http.py

```diff
@@ -44,25 +44,25 @@
 # Sidecar API
 # -----------
 @sidecar_api.post("/rewrite")
-async def http_rewrite(req: RewriteRequest) -> RewriteResponse:
+async def http_rewrite(req: RewriteRequest, request: Request) -> RewriteResponse:
```

Comment:
What is the second `request` parameter and why do we need it? It seems we don't use it anyway

Reply:
Good catch. We don't need it anymore. Removing....